packages/metascraper-feed/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const { toRule, url } = require('@metascraper/helpers')
+
+const toUrl = toRule(url)
+
+module.exports = () => {
+  return {
+    feed: [
```

Comment:
`feed` or `rss`? 🤔

Reply:
it isn't always an RSS feed. could be an atom feed or json..